test/e2e/fixture-project/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "private": true,
+  "dependencies": {
+    "@0no-co/graphqlsp": "file:../../..",
+    "graphql-tag": "^2.12.6"
```

Comment:
We can avoid a new dep by using urql/core

Reply:
I added pnpm workspaces, but unfortunately the plugin resolution of TS did not work when I used `"@0no-co/graphqlsp": "workspace:*"`. Probably some symlinking issues